### PR TITLE
Bifrost calib

### PR DESCRIPTION
**Pull request by mortenjc**
### Issue reference / description

The branch you merge from should already reference an event-formation-unit github ticket number. You can add a descriptive title, but if an issue is referenced, you don't have to.

## Checklist for submitter

- [ ] Check for conflict with integration test
- [ ] Unit tests pass

---

### Nominate for Group Code Review

- [ ] Nominate for code review

### CODE REVIEW

**Comment by mortenjc** [comment]
Caen calibration test fixes.

**Review by JennyLouise — APPROVED** [review]
Looks good! I'm not sure if you want to merge this in right now after the conversation we just had about longer term caen calibration plans so I'll leave that up to you!

**Pull request merged**
